[2023-08-28 20:09:21,387] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:09:21,416] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:09:21,417] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:09:21,418] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:09:21,419] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:09:21,477] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:09:21,531] {standard_task_runner.py:52} INFO - Started process 25332 to run task
[2023-08-28 20:09:21,556] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp9pnsskja', '--error-file', '/tmp/tmpm3nbct82']
[2023-08-28 20:09:21,568] {standard_task_runner.py:80} INFO - Job 15: Subtask cria_pasta
[2023-08-28 20:09:21,900] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:09:22,064] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:09:22,067] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:09:22,069] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:09:22,156] {subprocess.py:85} INFO - Output:
[2023-08-28 20:09:22,203] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:09:22,297] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T230921, end_date=20230828T230922
[2023-08-28 20:09:22,360] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:09:22,456] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:18:27,009] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:18:27,075] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:18:27,075] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:18:27,075] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:18:27,076] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:18:27,120] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:18:27,175] {standard_task_runner.py:52} INFO - Started process 28965 to run task
[2023-08-28 20:18:27,198] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmp5c01vi5i', '--error-file', '/tmp/tmp9w_3wa2z']
[2023-08-28 20:18:27,203] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-08-28 20:18:27,356] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:18:27,505] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:18:27,509] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:18:27,511] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:18:27,607] {subprocess.py:85} INFO - Output:
[2023-08-28 20:18:27,684] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:18:27,791] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T231827, end_date=20230828T231827
[2023-08-28 20:18:27,872] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:18:28,093] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:22:06,807] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:22:06,843] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:22:06,844] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:22:06,844] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:22:06,844] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:22:06,902] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:22:06,979] {standard_task_runner.py:52} INFO - Started process 30765 to run task
[2023-08-28 20:22:07,021] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpqvgv9xg5', '--error-file', '/tmp/tmpvvmyrcs3']
[2023-08-28 20:22:07,031] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-08-28 20:22:07,195] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:22:07,315] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:22:07,318] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:22:07,322] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:22:07,389] {subprocess.py:85} INFO - Output:
[2023-08-28 20:22:07,423] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:22:07,485] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T232206, end_date=20230828T232207
[2023-08-28 20:22:07,517] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:22:07,595] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:28:12,499] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:28:12,545] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:28:12,546] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:28:12,546] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:28:12,547] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:28:12,619] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:28:12,703] {standard_task_runner.py:52} INFO - Started process 579 to run task
[2023-08-28 20:28:12,751] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpvpx3krg5', '--error-file', '/tmp/tmpv8nnaffi']
[2023-08-28 20:28:12,765] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-08-28 20:28:13,078] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:28:13,431] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:28:13,436] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:28:13,443] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:28:13,544] {subprocess.py:85} INFO - Output:
[2023-08-28 20:28:13,611] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:28:13,728] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T232812, end_date=20230828T232813
[2023-08-28 20:28:13,816] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:28:14,057] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:30:26,366] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:30:26,392] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:30:26,392] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:30:26,393] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:30:26,393] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:30:26,431] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:30:26,478] {standard_task_runner.py:52} INFO - Started process 2068 to run task
[2023-08-28 20:30:26,495] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpsvvisfd0', '--error-file', '/tmp/tmpik714v0f']
[2023-08-28 20:30:26,501] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-08-28 20:30:26,635] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:30:26,780] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:30:26,783] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:30:26,785] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:30:26,868] {subprocess.py:85} INFO - Output:
[2023-08-28 20:30:26,937] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:30:27,033] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T233026, end_date=20230828T233027
[2023-08-28 20:30:27,081] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:30:27,193] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-28 20:40:44,300] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:40:44,328] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [queued]>
[2023-08-28 20:40:44,329] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:44,329] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-28 20:40:44,329] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-28 20:40:44,393] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-08-14 00:00:00+00:00
[2023-08-28 20:40:44,460] {standard_task_runner.py:52} INFO - Started process 4850 to run task
[2023-08-28 20:40:44,482] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta', 'scheduled__2023-08-14T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpvducarn9', '--error-file', '/tmp/tmpy6k9o0qf']
[2023-08-28 20:40:44,490] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-08-28 20:40:44,752] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta scheduled__2023-08-14T00:00:00+00:00 [running]> on host FelipeBreve-PC.localdomain
[2023-08-28 20:40:45,085] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-08-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-14T00:00:00+00:00
[2023-08-28 20:40:45,103] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-28 20:40:45,112] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/felipe/datapipeline/semana=2023-08-21"']
[2023-08-28 20:40:45,233] {subprocess.py:85} INFO - Output:
[2023-08-28 20:40:45,303] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-28 20:40:45,412] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta, execution_date=20230814T000000, start_date=20230828T234044, end_date=20230828T234045
[2023-08-28 20:40:45,461] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-28 20:40:45,597] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
